### ISSUE TITLE
Hello, the author, I am in training loss curve oscillation is very serious, is it a normal phenomenon?

### ISSUE THREAD

**Issue by wen-flow** [MVIG-SJTU/AlphAction#35]
Hello, the author, I am in training loss curve oscillation is very serious, is it a normal phenomenon?

**Comment by yelantf** [comment]
How large batch size are you using? When training with small batch sizes, the curve oscillation is a bit serious.

**Comment by wen-flow** [comment]
Thank you for your reply. As I have only one 2080ti video card, I can only set the batchsize to 2,I would also like to ask whether the open source code provides visualization of output results or only output test indicators?

**Comment by yelantf** [comment]
The `test_net.py` script can only be used to output test indicators (mAP). You can use the `demo/demo.py` for visualizing results.

**Comment by yelantf** [comment]
I am closing this issue since your question has already been answered.
